README.md

```diff
@@ -1,4 +1,681 @@
-# eventstreams-go-sdk
-Event Streams SDK allows you to integrate your application with Event Streams RESTful APIs.
+[![Build Status](https://travis-ci.com/IBM/adminrest-go-sdk.svg?token=eW5FVD71iyte6tTby8gr&branch=main)](https://travis.ibm.com/IBM/adminrest-go-sdk.svg?token=eW5FVD71iyte6tTby8gr&branch=main)
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
-Coming soon!
+# Eventstreams_Go_SDK 0.0.1
+Go client library to interact with the various [IBM Cloud Eventstreams APIs](https://cloud.ibm.com/apidocs?category=eventstreams_sdkv1).
+
+Disclaimer: this SDK is being released initially as a **pre-release** version.
+Changes might occur which impact applications that use this SDK.
+
+## Table of Contents
+<!--
+  The TOC below is generated using the `markdown-toc` node package.
+
+      https://github.com/jonschlinkert/markdown-toc
+
+  You should regenerate the TOC after making changes to this file.
+
+      npx markdown-toc -i README.md
+  -->
+
+<!-- toc -->
+
+- [Overview](#overview)
+- [Prerequisites](#prerequisites)
+- [Installation](#installation)
+    + [`go get` command](#go-get-command)
+    + [Go modules](#go-modules)
+    + [`dep` dependency manager](#dep-dependency-manager)
+- [Using the SDK](#using-the-sdk)
+- [Questions](#questions)
+- [Issues](#issues)
+- [Open source @ IBM](#open-source--ibm)
+- [Contributing](#contributing)
+- [License](#license)
+
+<!-- tocstop -->
+
+## Overview
+
+The IBM Cloud Eventstreams Go SDK allows developers to programmatically interact with the following IBM Cloud services:
+
+Service Name | Package name 
+--- | --- 
+<!-- [Example Service](https://cloud.ibm.com/apidocs/example-service) | exampleservicev1 -->
+
+## Prerequisites
+
+[ibm-cloud-onboarding]: https://cloud.ibm.com/registration
+
+* An [IBM Cloud][ibm-cloud-onboarding] account.
+* An IAM API key to allow the SDK to access your account. Create one [here](https://cloud.ibm.com/iam/apikeys).
+* Go version 1.14 or above.
+
+## Installation
+The current version of this SDK: 0.0.1
+
+There are a few different ways to download and install the Eventstreams Go SDK project for use by your
+Go application:
+
+#### `go get` command  
+Use this command to download and install the SDK to allow your Go application to
+use it:
+
+```
+go get -u github.com/IBM/adminrest-go-sdk
```

Comment:
Isn't it `github.com/IBM/eventstreams-go-sdk`?

Reply:
fixed